package.json

```
@@ -52,25 +52,23 @@
     "type-check": "loom type-check",
     "test": "loom test",
     "check": "npm-run-all lint type-check test",
-    "clean": "rimraf build build-internal \"./src/styles/polaris-tokens\"",
+    "clean": "rimraf build build-internal",
     "build": "node ./scripts/build.js",
     "prebuild-consumer": "yarn run build",
     "build-consumer": "node ./scripts/build-consumer",
     "secrets": "ejson decrypt -o secrets.json secrets.ejson",
     "prenew-version-pr-generator": "rimraf sandbox && yarn run secrets",
     "new-version-pr-generator": "node ./scripts/new-version-pr-generator",
-    "copy-polaris-tokens": "rimraf ./src/styles/polaris-tokens && shx cp -r ./node_modules/@shopify/polaris-tokens/dist ./src/styles/polaris-tokens",
     "prepublishOnly": "yarn run build",
-    "dev": "npm-run-all copy-polaris-tokens storybook",
+    "dev": "yarn run storybook",
     "start": "serve ./build-internal/storybook/static -l ${PORT:=6006}",
     "readme-update-version": "node ./scripts/readme-update-version",
     "version": "yarn run readme-update-version",
     "storybook": "start-storybook -p 6006 --quiet",
-    "storybook:build": "yarn run copy-polaris-tokens && build-storybook -o build-internal/storybook/static"
+    "storybook:build": "yarn run build-storybook -o build-internal/storybook/static"
```

Stuff in the node_modules/.bin folder is already in the binpath, so you don't need the `yarn run` bit

```suggestion
    "storybook:build": "build-storybook -o build-internal/storybook/static"
```